### PR TITLE
Fixed a bug with rendering response containing jsut StatusCode

### PR DESCRIPTION
**Pull request by kzangeli**
Fixed a bug of rendering a StatusCode as complete response

### CODE REVIEW

File: src/lib/serviceRoutines/postIndividualContextEntityAttribute.cpp [hunk 1]
```diff
@@ -61,7 +61,8 @@ std::string postIndividualContextEntityAttribute
                                                                 &parseDataP->upcar.res,
```
Comment (fgalan) [inline src/lib/serviceRoutines/postIndividualContextEntityAttribute.cpp]: CHANGE_NEXT_RELEASE entry missing.

Maybe should add some test harness to check this?

Comment (kzangeli) [inline src/lib/serviceRoutines/postIndividualContextEntityAttribute.cpp]: Sure x 2

Comment (kzangeli) [inline src/lib/serviceRoutines/postIndividualContextEntityAttribute.cpp]: CHANGE_NEXT_RELEASE fixed in a51056f.

Comment (kzangeli) [inline src/lib/serviceRoutines/postIndividualContextEntityAttribute.cpp]: FUNC TEST added in 7d7fd3c

**Comment by fgalan** [comment]
LGTM

**Pull request merged**